npc/quests/quests_moscovia.txt

```diff
@@ -8624,10 +8624,10 @@ OnTouch:
 			mes "- You hear a splashing as -";
 			mes "- something gleaming -";
 			mes "- seems to stare at you!! -";
-			next;
+			close2;
 			specialeffect2 EF_FLASHER;
 			percentheal -100,0;
-			close;
+			end;
```

Comment:
maybe `unitkill getcharid(3);` would be better/suitable for killing a char? 
Perhaps would be helpful in future code standardization?